src/cljs/rems/config.cljs

```diff
@@ -19,7 +19,7 @@
  (fn [db _]
    (:config db)))
 
-(defn fetch-config! []
+(defn ^:export fetch-config! []
```

Comment:
my original implementation idea was to convert current config object to `js` and return it to the caller. unfortunately doing a `js->clj` resulted in some nasty error that i wasn't able to comprehend, and exporting `fetch-config!` worked as intended (resetting client configuration).